internal/relay/config.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/neutron-org/neutron-query-relayer/internal/config"
 )
 
-func GetNeutronChain(logger *zap.Logger, cfg *config.NeutronChainConfig) (*relayer.Chain, error) {
+func GetNeutronChain(logger *zap.Logger, cfg *config.NeutronChainConfig, accountPrefix string) (*relayer.Chain, error) {
```

Comment:
I dont think it makes sense to pass the prefix as variable, you can use `neutronapp.Bech32MainPrefix` as is, during  `cosmos.CosmosProviderConfig` instantiating

Reply:
Yeah, okay, thought about that, but wasn't sure